app/lib/import_export.rb

```diff
@@ -710,9 +710,11 @@ def extract_assessments_details(patients_identifiers, assessments, fields)
 
     assessments_details = []
     assessments.each do |assessment|
-      assessment_details = assessment.custom_details(fields, patients_identifiers[assessment.patient_id])
+      assessment_details = assessment.custom_details(fields, patients_identifiers[assessment.patient_id]) || {}
```

Comment:
In what case would `assessment.custom_details` return nil and fall back to `... || {}`? It looks like with this method it will always at least return `{}` if no fields are available. https://github.com/SaraAlert/SaraAlert/blob/3d935440ea0a8edea038fcde76a5a054901a69fc/app/models/assessment.rb#L141

Reply:
With how it is written now, no case. This was more of a check I put in in case custom_details was ever changed improperly and we wanted to avoid an error.

Reply:
Ok, that sounds fine